docker/Dockerfile.ubuntu

```diff
@@ -24,5 +24,3 @@ WORKDIR /holochain
 
 # run a build here to cache dependencies in a docker layer
 RUN cargo build
```

Comment:
Why does this have to happen in this image? Do I have to recompile the whole image when running tests?

Reply:
@sphinxc0re this make sure all the dependencies are already in the image that gets pushed to docker hub, so when you use `Dockerfile.test` to run tests, you don't have to wait for cargo

Reply:
Nice! 😄